tests/testthat/_snaps/tbl_survfit_quantiles.md

```diff
@@ -0,0 +1,65 @@
+# tbl_survfit_quantiles() works
+
+    Code
+      as.data.frame(tbl)
+    Output
+                        **Placebo**  \nN = 86 **Xanomeline High Dose**  \nN = 84 **Xanomeline Low Dose**  \nN = 84
+      1   Time to event                  <NA>                               <NA>                              <NA>
+      2          Median                    NA                               36.0                              33.0
+      3          95% CI              (NA, NA)                       (25.0, 47.0)                      (28.0, 51.0)
```

Comment:
The tests are very good imo! I have only a curiosity: let's say we have NA lines `(NA, NA)` like here; how can I make it into a single `NE`? Should we test these NA label changes?

Reply:
The function has an argument that accepts a function that will be used to round/style the statistics. The default function returns NAs, but we do have this open issue about whether we need something else that would 
allow us to change how NAs are returned. https://github.com/insightsengineering/crane/issues/28

Reply:
Ah that would be a nice addition imo!